src/config.ts

```diff
@@ -73,14 +102,19 @@ export const rollupChain = defineChain({
     ...chainConfig.contracts,
     l2OutputOracle: {
       [parentChain.id]: {
-        address: import.meta.env.VITE_L2_OUTPUT_ORACLE_ADDRESS,
+        address: '0x74Ad6E0FB793eB5e6c1ff1225B03F5C5fFB7EF0c',
       },
     },
     portal: {
       [parentChain.id]: {
-        address: import.meta.env.VITE_L1_OPTIMISM_PORTAL_ADDRESS,
+        address: '0x0d605bb7d4FB586eAB750205F5247825F4D8AF4B',
```

Comment:
These should not be set directly in code and should use the variables set in the `.env` file.